website/docs/r/cm_security_group_rule.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: security_group_rule"
+sidebar_current: "docs-rightscale-resource-security_group_rule"
+description: |-
+  Create and maintain a rightscale cloud management security_group_rule.
```

Comment:
Description should probably be consistent across resources. No need to specify "cloud management". Should be `RightScale`.

Reply:
I'll cleanup and standardize in a ticket tomorrow since this is across all resources.  CURRENTLY the above convention is consistent, although not in desired format.  ;-p